charts/k8ssandra/templates/stargate/service.yaml

```diff
@@ -4,6 +4,8 @@ kind: Service
 metadata:
   name: {{ .Release.Name }}-{{ include "k8ssandra.datacenterName" . }}-stargate-service
   labels:
+    release: {{ .Release.Name }}
```

Comment:
Just an FYI, with the chart refactoring we are standardizing the label templates. In #280 I have this:

```yaml
labels: {{ include "k8ssandra-common.labels" . | indent 4 }}
```
You don't need to worry about making the change here. I just wanted to point it out.

Reply:
Yeah, I made these changes before I had seen #280. :)